modules/aks/azure/variables.tf

```diff
@@ -69,3 +69,21 @@ variable "network_policy" {
   type        = string
   default     = null
 }
+
+variable "node_pools" {
+  default     = {}
+  description = "List of node pools with minimal configuration"
+  # type = map(map(object({
+  #   vm_size             = string
+  #   node_count          = number
+  #   availability_zones  = bool
+  #   enable_auto_scaling = bool
+  #   max_count           = number
+  #   min_count           = number
+  #   max_pods            = number
+  #   node_tains          = list(string)
+  #   os_disk_size_gb     = number
+  #   os_type             = string
+  #   vnet_subnet_id      = string
+  # })))
```

Comment:
Why is this commented?

Reply:
Misinterpreted why those lines were commented in the original code.
Still working on this one..